app/scripts/components/data-catalog/index.tsx

```diff
@@ -103,9 +94,16 @@ export const sortOptions = [{ id: 'name', name: 'Name' }];
 
 export interface DataCatalogProps {
   datasets: DatasetData[];
+  cardComponentOverride?: { // Allow user to override with a different cardType with optional props
+    CardComponent: ComponentType<CardComponentProps>;
+    props: Partial<CardComponentProps>;
+  } 
```

Comment:
📓 : Do we care to have the ability to override the card component? I've added this but i'm thinking we dont currently need it. If we do want to override card logic or styles, we can defer till we have that usecase and need to solve for. thoughts?

Reply:
I think we discussed this issue before, but to make it as a record: I don't think we need to offer ways of overriding specific components inside of the feature components (at lest for now.) If this is needed, a user can make their own list without overriding the specific components inside of the feature.

Reply:
I think this makes sense, if the user wants alot more flexibility or to use whatever card component they like, they will have the flexibility now to do this in their page views. i'll go ahead and remove